index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website: Ademola Adebayo</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <section class="toolbar">
+    <div class="top-toolbar">
+      <p>10:30</p>
+      <div class="top-icons">
+        <i class="	fas fa-signal fa-0.5x"></i>
+        <i class="	fas fa-wifi fa-0.5x"></i>
+        <i class="	fa fa-battery-0 fa-0.5x"></i>       
+      </div>
+    </div>
+    <div class="btm-toolbar">
+      <h2>Smart City</h2>
```

Comment:
- [ ] Please wrap this `h2` logo inside of an `a` tag. This is considered good practice to wrap logos inside of `a` tags to act as quick access to the home page.